src/borg/crypto/low_level.pyx

```diff
@@ -184,10 +184,10 @@ class UNENCRYPTED:
 
 
 cdef class AES256_CTR_BASE:
-    # Layout: HEADER + MAC 32 + IV 8 + CT (same as attic / borg < 1.2 IF HEADER = TYPE_BYTE, no AAD)
+    # Layout: HEADER + MAC 32 + IV 8 + CT (same as attic / borg < 1.3 IF HEADER = TYPE_BYTE, no AAD)
 
     cdef EVP_CIPHER_CTX *ctx
-    cdef unsigned char *enc_key
+    cdef unsigned char enc_key[32]
```

Comment:
we were lucky that worked all the time, see "avoid losing the key" commit.

Reply:
i filed an issue to fix this also in the other branches, so i can close this here.